From 6e486e9d3c2335e13ce93b5cbaea379f055b3223 Mon Sep 17 00:00:00 2001
From: Thomas Koeller <thomas.koeller@baslerweb.com>
Date: Thu, 5 Nov 2020 11:45:25 +0100
Subject: [PATCH 32/34] Reformatted Makefile

Reformatted the DTS Makefile for uniform coding style. This change is
purely cosmetic.
---
 arch/arm64/boot/dts/freescale/Makefile | 18 +++++++-----------
 1 file changed, 7 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 26da54c6d7b9..9320e0f39ce2 100755
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -164,15 +164,11 @@ dtb-$(CONFIG_ARCH_MXC) += imx8dxl-evk.dtb imx8dxl-evk-rpmsg.dtb \
 			  imx8dxl-evk-root.dtb imx8dxl-evk-inmate.dtb
 dtb-$(CONFIG_ARCH_S32) += s32v234-evb.dtb \
 			  s32v234-sbc.dtb
-
-
-dtb-$(CONFIG_ARCH_MXC) += \
-	imx8mm-evk-basler-camera.dtb \
-	imx8mm-ddr4-evk-basler-camera.dtb \
-	imx8mm-evk-basler-camera-1lane.dtb \
-	imx8mm-ddr4-evk-basler-camera-1lane.dtb
-dtb-$(CONFIG_ARCH_MXC) += \
-	imx8mq-evk-basler-camera.dtb \
-	imx8mq-evk-basler-camera-csi1.dtb \
-	imx8mq-evk-basler-camera-csi2.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-evk-basler-camera.dtb \
+			  imx8mm-ddr4-evk-basler-camera.dtb \
+			  imx8mm-evk-basler-camera-1lane.dtb \
+			  imx8mm-ddr4-evk-basler-camera-1lane.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mq-evk-basler-camera.dtb \
+			  imx8mq-evk-basler-camera-csi1.dtb \
+			  imx8mq-evk-basler-camera-csi2.dtb
 
-- 
2.30.0

